PacificPower/readPP.js

```diff
@@ -527,6 +555,37 @@ let page = "";
     }
   }
 
+  const pacificPowerMeters = "pacific_power_data";
+
+  for (let i = 0; i < PPArray.length; i++) {
+    console.log(PPArray[i]);
+
+    // Comment out the axios POST request as specified below for local development (unless making changes to upload stuff).
+    // Uncomment this section before pushing to production.
```

Comment:
Not high priority, but it could be a good idea to use command line arguments in the future: https://github.com/OSU-Sustainability-Office/automated-jobs/issues/41